### PR TITLE
C64 soft80 cpeek stuff

### PR DESCRIPTION
**Pull request by mrdudz**
up for discussion, not 100% ready yet (maybe someone wants to test)

### CODE REVIEW

**Comment by mrdudz** [comment]
ok working now, unless someone finds a problem, ready to merge :)

File: testcode/lib/cpeek-test.c [hunk 1]
```diff
@@ -141,8 +152,8 @@ static unsigned char testCPeekC (char ch)
     */
     ch2_c = peekChWithoutTranslation ();
     if ((ch2_c != ch2_b)
-#if defined(__C128__)
-        /* VDC memory is not accessable */
+#if defined(__C128__) || defined(__C64__)
+        /* VDC memory is not accessable, soft80 has no "videoram" */
```
Comment (polluks) [inline testcode/lib/cpeek-test.c]: accessible

File: libsrc/c64/extra/soft80.s [hunk 2]
```diff
@@ -50,3 +66,6 @@
         ; Chars used by chline () and cvline ()
         .exportzp       chlinechar = CH_HLINE
         .exportzp       cvlinechar = CH_VLINE
+
+        .import         return1
+        .export         _doesclrscrafterexit := return1
```
Comment (oliverschmidt) [inline libsrc/c64/extra/soft80.s]: Nice idea to have a soft80-specific doesclrscrafterexit() !

**Comment by oliverschmidt** [comment]
Fixed with https://github.com/cc65/cc65/commit/040134e7750f81a6a3b54856c4a1bf38d9244b5c.

**Comment by mrdudz** [comment]
\o/

**Pull request merged**